File: app/src/main/java/com/guideapp/guideapp/sync/GuideSyncIntentService.java
package com.guideapp.guideapp.sync;

import android.app.IntentService;
import android.content.Intent;

public class GuideSyncIntentService extends IntentService {

    public GuideSyncIntentService() {
        super("GuideSyncIntentService");
    }

    @Override
    protected void onHandleIntent(Intent intent) {
        GuideSyncTask.syncGuide(this);
    }
}

File: app/src/main/java/com/guideapp/sync/GuideSyncIntentService.kt
package com.guideapp.sync

import android.app.IntentService
import android.content.Intent

class GuideSyncIntentService : IntentService("GuideSyncIntentService") {
    override fun onHandleIntent(intent: Intent?) {
        GuideSyncTask.syncGuide(this)
    }
}